landscaper/gardenlet/example/landscaper-resources/installation/component-descriptor.yaml

```diff
@@ -0,0 +1,48 @@
+# Registry: stored under component.baseUrl + component.name + component.version
+# eu.gcr.io/gardener-project/landscaper/dev/components/github.com/gardener/gardener:latest
+
+# {
+#     "mediaType": "application/sap-cnudie+tar",
+#     "digest": "",
+#     "size": 123
+#  }
+
```

Comment:
```suggestion
```

Reply:
At least for me, that comment helps to better understand the concepts, plus it is only a comment in the example folder. But of course I can remove it.

Reply:
No strong opinion, you can also keep it. Generally, I personally don't use such comments if the contract is not maintained in the repo itself, i.e., when this changes then somebody would have to adapt this example here and nobody will do it.

Reply:
ok makes sense.